config/jobs/kubernetes-csi/external-attacher/external-attacher-config.yaml

```diff
@@ -0,0 +1,214 @@
+# generated by gen-jobs.sh, do not edit manually
+
+presubmits:
+  kubernetes-csi/external-attacher:
+  - name: pull-sig-storage-external-attacher-1-13-on-kubernetes-1-13
```

Comment:
For sidecar presubmits, I think we only need the following jobs:
* canary on K8s master
* canary on K8s master - 1